src/components/breadcrumb/SubgraphBreadcrumbItem.vue

```diff
@@ -120,7 +121,27 @@ const menuItems = computed<MenuItem[]>(() => {
       command: async () => {
         await workflowService.duplicateWorkflow(workflowStore.activeWorkflow!)
       },
-      visible: props.item.key === 'root'
+      visible: isRoot
+    },
+    {
+      separator: true,
+      visible: isRoot
+    },
+    {
+      label: t('menuLabels.Save'),
+      icon: 'pi pi-save',
+      command: async () => {
+        await useCommandStore().execute('Comfy.SaveWorkflow')
+      },
+      visible: isRoot
+    },
+    {
+      label: t('menuLabels.Save As'),
+      icon: 'pi pi-save',
+      command: async () => {
+        await useCommandStore().execute('Comfy.SaveWorkflowAs')
+      },
+      visible: isRoot
```

Comment:
Tried to make a single suggestion, but GitHub won't let me.

How do you feel about this? It lets you use the property shorthand.

```suggestion
      visible
    },
    {
      separator: true,
      visible
    },
    {
      label: t('menuLabels.Save'),
      icon: 'pi pi-save',
      command: async () => {
        await useCommandStore().execute('Comfy.SaveWorkflow')
      },
      visible
    },
    {
      label: t('menuLabels.Save As'),
      icon: 'pi pi-save',
      command: async () => {
        await useCommandStore().execute('Comfy.SaveWorkflowAs')
      },
      visible
```